source/query/filter.txt

```diff
@@ -364,6 +364,35 @@ The query returns the following document:
       "dateCreated": { "$date": "2003-03-26" }
    }
 
+Match by Nested Field
```

Comment:
Generally we've been trying to use "embedded" as opposed to "nested" in our docs.